doc/zettelkasten.txt

```diff
@@ -258,6 +278,72 @@ get_toc({note_id}, {format})                               *zettelkasten-get_toc
             range = true,
         })
 
+contains({path})                               *zettelkasten-is_in_notes_path*
+
+    Determines whether the given {path} is withing the directory structure
+    rooted at {notes_path}. 
+
+    This is also included as `v:lua.zettelkasten.contains`
+
+    Parameters: ~
+        {path} (string): The path to test for.
+
+    Returns: ~
+        {boolean}: `true` if {path} is within {notes_path} `false` otherwise.
+
+    Example: ~
+    When selectively setting options for the zettelkasten in the markdown
+    ftplugin: >
+
+    if v:lua.zettelkasten.contains(expand('%'))
```

Comment:
This is a good example, maybe you should add it to `ftplugin/markdown.lua`.

Reply:
Thank you. I agree, but I am not sure what, if anything, it would replace: 
Should the entire current markdown.lua be wrapped in a `contains`?